proposals/0033-data-collections.md

```diff
@@ -0,0 +1,244 @@
+- Start Date: 2023-05-01
+- Reference Issues: https://github.com/withastro/roadmap/issues/530
+- Implementation PR: https://github.com/withastro/astro/pull/6850
+
+# Summary
+
+Introduce a standard to store data separately from your content (ex. JSON files).
+
+# Example
+
+Like content collections, data collections are created in the `src/content/` directory. These collections should include only JSON files:
+
+```
+src/content/
+  authors/
+    ben.json
+    tony.json
+```
+
+These collections are configured using the same `defineCollection()` utility in your `content/config.ts` with `type: 'data'` specified:
+
+```ts
+// src/content/config.ts
+import { defineCollection, z } from 'astro:content';
+
+const authors = defineCollection({
+  type: 'data',
+  schema: z.object({
+    name: z.string(),
+    twitter: z.string().url(),
+  })
+});
+
+export const collections = { authors };
+```
+
+These collections can also be queried using the `getCollection()` and `getEntry()` utilities:
```

Comment:
You mention `getEntries()` in the video, but I did a quick search on the page and didn't see that here. Is that included or not? If included, it should be defined here!

Reply:
Good point! We should add that 👍

Reply:
Update: added to the references doc. `getEntries()` is meant for resolving references primarily and probably shouldn't be recommended for manual use. In fact, we should add a documentation note that "if you're calling `getEntries()` on its own, you're probably looking for `getCollection()`"